.gitattributes

```diff
@@ -1 +0,0 @@
-test/** linguist-generated
```

Comment:
Is there a reason to remove this?

Reply:
@terrytangyuan Since I deleted legacy ksonnet test files, this check isn't longer needed.